public/assets/css/contact.css

```diff
@@ -20,32 +20,36 @@ label {
   display: block;
 }
 
-
+input[type=submit] {
```

Comment:
use className instead otherwise all the submit input of the page will take this behavior (can be an issue with header searchbar or future implementation)